### PR TITLE
Drop UI general / QoL improvements

### PR DESCRIPTION
**Pull request by Senexis**
## Contents
Features:
- Allow searching in items only, locations only, or items and locations (useful when looking relics or what relics drop)
- Allow sorting the different table columns in both ascending and descending orders with indicator icons

Fixes:
- Fix the build time not being /that/ legible to most people (toLocaleString makes more sense than toISOString)
- Fix Cetus bounty missions being listed inconsistently with other missions (allows it to show up when searching for "Earth")
- Fix needing to increment the script version in 2 places to make it go through the incredibly complicated upgrade process

## Notes
This is my personal preference of style. If you wish, you could edit the style to match your preference. I have tested this relatively thoroughly and it works, though you might want to test this as well. I could have missed some things, after all.

Merging this would close issues #5 and #6.

### CODE REVIEW

**Comment by TobiTenno** [comment]
do you have it deployed anywhere so we could see it?

**Comment by Senexis** [comment]
Sorry! I used a local web server to develop, so unfortunately, no.

**Comment by TobiTenno** [comment]
Seems great, but doesn't allow sorting rarity

**Comment by Senexis** [comment]
True. The thing is that rarity directly correlates to chance. When testing, I personally found it confusing to click "sort by rarity" only to find it sorting by chance... It seems kind of unnecessary to have both columns sort-able when they have the same end result.

Would be easy to fix if needed, just doesn't seem all that necessary however.

**Comment by TobiTenno** [comment]
hmmm, i think it's fine as-is. i want to let atomic review, tho, but i'm good with the update, it's nice.

**Comment by atomicptr** [comment]
It breaks the open search support as it is right now. Though you probably only have to add "/both" to that to make it work as intended again. If you add that we can merge this, good job 👍 .

https://github.com/WFCD/warframe-drop-data/blob/gh-pages/opensearch.xml

File: index.html [hunk 1]
```diff
@@ -179,12 +190,13 @@ <h5>Warframe Drop Data</h5>
         let info = JSON.parse(localStorage.getItem("_wfinfo"))
         let date = new Date(info.timestamp)
 
-        $("#lastbuildtime").text(date.toISOString())
+        $("#lastbuildtime").text(date.toLocaleString())
 
-        let matches = window.location.hash.match(/\#?\/search\/(.*)\//)
+        let matches = window.location.hash.match(/\#?\/search\/(.*)\/(items|locations|both)/)
```
Comment (atomicptr) [inline index.html]: This should also accept not having "items", "locations" or "both" (maybe both if nothing?), because of A) old links and B) open search

Comment (Senexis) [reply index.html]: All done in 2192f5b70930412db010d168ecd02a44ada5f939!

**Comment by Senexis** [comment]
Note that the fix doesn't enforce "search" in the URL. If later on you're going to need more hash locations, this might have to be enforced. If this is an issue let me know.

**Comment by atomicptr** [comment]
Looks good, good job 👍

**Pull request merged**